css/style.css

```diff
@@ -147,6 +197,48 @@ header .logo .logo-title {
     color: #344563;
     text-align: start;
   }
+
+  .btn {
+    width: auto;
+    padding: 5px;
+    border-radius: 8px;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #396df2;
+    border: 1px solid #396df2;
+    background-color: transparent;
+  }
+
+  .categories ul li {
+    border-bottom: 1px solid #dfe1e6;
+    padding-top: 6px;
+    padding-bottom: 6px;
+  }
+
+  .categories ul li .text {
+    display: inline-block;
```

Comment:
- [ ] It would be better if we implement the CSS DRY principle, let us avoid repeating style rules, it makes our codebase messy and hard to debug